### ISSUE TITLE
plantum.jar file location.

### ISSUE THREAD

**Issue by AtosNicoS** [peele/plantuml-preview#37]
Hi,
I am trying to use this atom package to render my plantuml diagram. I've placed the plantuml.jar file directly in the root of my git, where also my .pu files are currently located. The problem is that plantuml-preview does not recognize this .jar file.

If I use an absolut path, it is recognized correct:
C:\git\<project>\plantuml.jar

Am I missunderstanding something? Cant I use a relative location to the docfile itself? I plan to always ship a .jar file along with my docs, so it is simpler to regenerate them.

**Comment by peele** [comment]
Didn’t expect that use case. My view was the plugin configuration was own
by the atom installation and not based on project/working directory.

Seems straight forward to support looking for the jar based on current uml
file parent directory or maybe even project folder. Not sure how atom
handles things when you use the “open project” option. This would probably
still fallback to the global configured path if it fails.

Not really considering making all the plugin settings local the project.

**Comment by AtosNicoS** [comment]
Yes I also understood how it is intendet to be used. Maybe some more text as information should be added (either to the readme or to the setting menu itself). An example path for windows(`C:\Program Files\PlantUML\plantuml.jar`), and an example path for linux users (`/usr/bin/plantuml`) where to place the .jar file/where to look at. Same for the graphviz package.

About the relativ path: Now that I understood the intended usage, I am not sure if this is a good idea to use. However a relative path to the project/git root would make most sense i guess. If you have a general layout like `gitproject/doc/tools/plantuml.jar` it could work.

**Comment by naesean** [comment]
@peele 
I'm confused... where is the plantuml.jar file supposed to be located? There is nothing in the documentation and I'm getting an error: 
`plantuml-preview: plantuml.jar is not a file.`

**Comment by marcochin-tile** [comment]
@naesean Old comment, but for future reference for others. Download the plantuml jar from here https://plantuml.com/download and then just add the path to it on your computer in the package settings.
